plasma_framework/package.json

```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "test": "truffle test",
     "coverage": "truffle run coverage",
-    "docgen": "[[ -z $(git status --porcelain) ]] && rm -rf contracts/mocks && solidoc ./ ./docs && git checkout -- contracts/mocks || echo 'Please make sure all files are committed and the git status is clean' ",
+    "docgen": "bash docgen.sh",
```

Comment:
I guess we can just put `./docgen.sh` ? (not sure whether that `bash` would work on my side since I am using `zsh`...)

eg. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/package.json#L13
Meanwhile, shall we put it under `scripts/` ?

Reply:
done